content/applications/general/voip/devices_integrations.rst

```diff
@@ -0,0 +1,177 @@
+========================
+Devices and integrations
+========================
+
+:abbr:`VoIP (Voice over Internet Protocol)` can be used on many different devices, ranging from, a
+computer, tablet, mobile phone and many more. This is helpful in that it reduces costs, and
+employees can work from anywhere in the world with a broadband internet connection.
+
+Odoo *VoIP* is SIP (Session Initiation Protocol) compatible, which means that it can be used with
+*any* :abbr:`SIP (Session Initiation Protocol)` compatible application. This document covers setting
+up Odoo *VoIP* across different devices and integrations.
+
+Odoo is fully integrated with all Odoo apps, users can click into any app and schedule a call as an
+activity in the chatter.
+
+.. example::
+   For example, in the *CRM* app a user can click into an opportunity and click on
+   :guilabel:`Activities` in the chatter. Next, choose :guilabel:`Call` and under :guilabel:`Due
+   Date` select a date. Click :guilabel:`Save` and now in the chatter an activity shows up. Should
+   the :guilabel:`Due Date` be for today's date, then the activity shows up in the :abbr:`VoIP
+   (Voice over Internet Protocol)` widget.
+
+   .. image:: devices_integrations/crm-voip-widget.png
+      :align: center
+      :alt: View of CRM leads and the option to schedule an activity for Odoo Discuss
+
+   .. seealso::
+      :doc:`voip_widget`
+
+Odoo VoIP (laptop/desktop computer)
+===================================
+
+Odoo :abbr:`VoIP (Voice over Internet Protocol)` app and widget can be used from any browser on a
+laptop or desktop device. Simply click on the :guilabel:`📞 (phone)` icon in the upper right corner
+and the widget appears.
+
+.. seealso::
+   To see use of the :abbr:`VoIP (Voice over Internet Protocol)` widget on a desktop/laptop computer
+   see this documentation: :doc:`voip_widget`.
+
+Odoo VoIP (tablet/mobile device)
+================================
```

Comment:
in this section, I would make it clear that Odoo will *not* ring on your device for an incoming call; e.g.
```
Whilst you can place outgoing calls using Odoo on your mobile device,
be aware that Odoo is not a full VoIP application and will for example
not ring on incoming calls. If you want to be reachable on your mobile
device at all times, you should use an app like Zoiper which will stay
connected in the background at all times.
```
or words to that effect